src/App.tsx

```diff
@@ -53,7 +55,7 @@ function App() {
         },
     }, [client, unlinkContactFn]);
 
-    if (!client || isLoading) {
+    if (!client || isLoading && !context) {
```

Comment:
Do we need the context check? If `isLoading`, I'd expect a spinner?

Reply:
Removed _(sorry, I removed but forgot to comment)_